mlprimitives/timeseries.py

```diff
@@ -0,0 +1,63 @@
+import pandas as pd
+import numpy as np
+
+
+def create_window_sequences(df_timeseries, window_size):
+    """
+        Function that takes in a Pandas.DataFrame and a window_size then creates
+            output arrays that correspond to a timeseries sequence with window_size overlap.
+            The output arrays can be fed into a timeseries forecasting model.
+        Inputs:
+            df_timeseries (Pandas.DataFrame): a Pandas dataframe which has 'timestamp'
+                and 'value' columns, and is sorted based on timestamp. 
+                The timestamp column is in UNIX format (in seconds).
+            window_size (int): number of values that overlap to create the sequence.
+        Outputs:
+            x (numpy.ndarray): contains the time series sequenced data.
+            y (numpy.ndarray): acts as the label for the forecasting problem.
+            time (numpy.ndarray): the corresponding timestamps series.
+    """
+    X = []
+    Y = []
+    time = []
+    for i in range(len(df_timeseries) - window_size):
+        X.append(df_timeseries[i: i + window_size]['value'].values.copy().reshape([-1, 1]))
+        Y.append(df_timeseries[i + 1: i + window_size + 1]['value'].values.copy().reshape([-1, 1]))
```

Comment:
Is this implementation right? If I understand it properly, if window size is 50, each element appended to `X` will have 50 rows, which is right. But each element appended to `y` will ALSO have 50 rows, which I believe is incorrect.

Reply:
Good point. My model was returning sequences (in case we want to stack LSTM units). But I guess this shouldn't be the default for a primitive. I will add functionality to have a default parameter such that return_sequences is set to False.

Reply:
Or actually I will just make it return a single Y. I don't think it's necessary to have a sequence, even if we are doing a stacked LSTM (LSTM output is input to another LSTM).